simple-acquaintances/data/datagen.py

```diff
@@ -0,0 +1,386 @@
+"""
+Generates data for the simple-acquaintances PSL example.
+
+Use -h/--help for usage options.
+"""
+
+import argparse
+import json
+import numpy as np
+import os
+
+from itertools import combinations
```

Comment:
`combinations` is only used once.
Just import `itertools` can call it directly.